Libs/vtkSegmentationCore/vtkSegmentationConverter.h

```diff
@@ -55,10 +55,14 @@ class vtkSegmentationCore_EXPORT vtkSegmentationConverter : public vtkObject
 
   /// Default representation types
   /// In binary and fractional labelmaps values <=0 are considered background voxels (outside), values>0 are foreground (inside).
-  static const char* GetSegmentationBinaryLabelmapRepresentationName() { return "Binary labelmap"; };
+  static const char* GetSegmentationBinaryLabelmapRepresentationName()     { return "Binary labelmap"; };
   static const char* GetSegmentationFractionalLabelmapRepresentationName() { return "Fractional labelmap"; };
-  static const char* GetSegmentationPlanarContourRepresentationName() { return "Planar contour"; };
-  static const char* GetSegmentationClosedSurfaceRepresentationName() { return "Closed surface"; };
+  static const char* GetSegmentationPlanarContourRepresentationName()      { return "Planar contour"; };
+  static const char* GetSegmentationClosedSurfaceRepresentationName()      { return "Closed surface"; };
+  static const char* GetBinaryLabelmapRepresentationName()     { return GetSegmentationBinaryLabelmapRepresentationName(); };
```

Comment:
Maybe we should make the old methods deprecated? We should at least document which methods should be used, but even better, we could log a warning if the deprecated methods are used to discourage their usage.

Reply:
OK I can do it, but later (when I'm back) and in a separate commit, as these functions are called from several extensions as well, and I don't want to introduce warnings in them.
I did this because I realized that the "segmentation" in their name was unnecessary and they were super long.

Reply:
OK. For now, it is enough to add comments that tell which method names are preferred.